snippets/standard/datetime/json/datetime-converter-examples/example2/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Buffers;
+using System.Buffers.Text;
+using System.Diagnostics;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace DateTimeConverterExamples
+{
+    public class DateTimeConverterExample2 : JsonConverter<DateTime>
```

Comment:
nit: Instead of having names contain Example1/2, maybe make them descriptive? Up to you.

Maybe:
`DateTimeConverterCustomStandardFormat_R`